pack/fhv/fhvc.json

```diff
@@ -797,10 +1090,201 @@
+    "back_text": "Mirror Nest is connected to the location in the clockwise and counter-clockwise directions.",
+    "clues": 2,
+    "code": "10669",
+    "double_sided": true,
+    "encounter_code": "fate_of_the_vale",
+    "encounter_position": 22,
+    "faction_code": "mythos",
+    "illustrator": "Hai Hoang",
+    "name": "Mirror Nest",
+    "pack_code": "fhvc",
+    "position": 169,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "Mirror Nest is connected to the location in the clockwise and counter-clockwise directions.\n<b>Forced</b> - When you end your turn at this location: Discard 1 card at random from your hand.\n[action] Spend 7 [per_investigator] clues, as a group: Search The Abyss for a card and draw it. (Group limit once per act.)",
+    "traits": "Shattered. Lair.",
+    "type_code": "location"
+  },
+  {
+    "back_link": "10670b",
+    "clues": 2,
+    "code": "10670",
+    "encounter_code": "fate_of_the_vale",
+    "encounter_position": 23,
+    "faction_code": "mythos",
+    "illustrator": "Ethan Patrick Harris",
+    "name": "The Abyss",
+    "quantity": 1,
+    "pack_code": "fhvc",
+    "position": 170,
+    "shroud": 5,
+    "subname": "Spiraling Oblivion",
+    "text": "<b>Forced</b> - When you enter The Abyss, if there are cards in The Abyss: You are <b>killed</b>.\nDuring the enemy phase, each [[Shattered]] enemy at The Abyss loses aloof.\nThe Abyss always covers the top of The Abyss deck.",
+    "traits": "Extradimensional. Shattered.",
+    "type_code": "location"
+  },
+  {
+    "code": "10670b",
+    "encounter_code": "fate_of_the_vale",
+    "encounter_position": 23,
+    "faction_code": "mythos",
+    "hidden": true,
+    "name": "The Abyss",
+    "quantity": 1,
+    "pack_code": "fhvc",
+    "position": 170,
+    "text": "This card must always cover the top of The Abyss.\nCards are always drawn and revealed from the bottom of The Abyss. When a non-weakness encounter card is discarded, it is <b>always</b> placed on top of The Abyss.\n<b>Forced</b> - When you draw a card from The Abyss, if that card is ...\n- ... a treachery or an enemy: Draw it.\n- ... a [[Resident]] card: Flip it to its enemy side. It gains \"<b>Revelation</b> - This enemy attacks you <i>(without engaging you)</i>. Discard this enemy.\"\n- ... a player card: Add it to its owner's hand.\n- ...a location: Put it into play, unrevealed, adjacent to the nearest Mirror Nest (that is not already connected to a [[Cave]] location, if able). For the remainder of this scenario, the two locations are connected to one another.\n- ... any investigator's \"true\" investigator card: Its owner heals all damage and horror and takes control of it. Then, they flip their Shattered Self investigator card to its Old Memory side and put it into play under their control.",
```

Comment:
```suggestion
    "text": "This card must always cover the top of The Abyss.\nCards are always drawn and revealed from the bottom of The Abyss. When a non-weakness encounter card is discarded, it is <b>always</b> placed on top of The Abyss.\n<b>Forced</b> - When you draw a card from The Abyss, if that card is ...\n- ...a treachery or an enemy: Draw it.\n- ...a [[Resident]] card: Flip it to its enemy side. It gains \"<b>Revelation</b> - This enemy attacks you <i>(without engaging you)</i>. Discard this enemy.\"\n- ...a player card: Add it to its owner's hand.\n- ...a location: Put it into play, unrevealed, adjacent to the nearest Mirror Nest (that is not already connected to a [[Cave]] location, if able). For the remainder of this scenario, the two locations are connected to one another.\n- ...any investigator's \"true\" investigator card: Its owner heals all damage and horror and takes control of it. Then, they flip their Shattered Self investigator card to its Old Memory side and put it into play under their control.",
```

Reply:
don't know what's up with the highlighting here, seems like a GH issue?

Reply:
I applied the suggestion fwiw